### ISSUE TITLE
Generalization of Camera Stream in context of IP Cameras / Headless Jetson Nano

### ISSUE THREAD

**Issue by milindparikh** [maartenbreddels/ipywebrtc#61]
I have  been able to install ipywebrtc on a headless Jetson Nano with multiple IP Cameras (aka remote-cameras). I am accessing  the ipywebrtc through jupyter notebook on another machine (aka local-machine).

The network is dynamic (DHCP). Let's assume that the addressing the IP cameras and authenticating to IP cameras is a solved issue (auto-discovery through nmap, auth information stored elsewhere) 
 
In other words, 
    cap = cv2.VideoCapture("rtsp://user:password@xx.xx.xx.xxx:554/cam/realmonitor?channel=1&subtype=1") 
    ret, frame = cap.read() 
returns a good frame that can be displayed  through plt.imshow()

The CameraStream makes use of the specific camera installed on the local-machine. How to enable the use of remote cameras through the CameraStream?

**Comment by maartenbreddels** [comment]
There is no support for remote cameras directly, but you can stream it over
Weber,. See the chat room examples. Cheers

(from mobile phone)
